app/views/content_change_requests/_request_details.html.erb

```diff
@@ -55,7 +55,13 @@
 <%= render partial: "support/time_constraint", locals: { f: f } %>
 
 <div class="alert alert-info alert-block">
-  <p>If your request is urgent, please fill in this form and call <%= @primary_contact_details[:urgent_department_content_change][:phone] %> during office hours, or <%= @primary_contact_details[:urgent_mainstream_content_change][:phone] %> out of hours. (It's ONLY urgent when the public or the government is facing immediate and significant financial, legal or physical risk.)</p>
+  <p><strong>If your request is urgent</strong></p>
+  <p>Submit this form and call:</p>
+  <ul>
+    <li><a href="tel:+447810750834"><%= @primary_contact_details[:urgent_department_content_change][:phone] %></a> between 9am and 6pm</li>
+    <li><a href="tel:+447827992603"><%= @primary_contact_details[:urgent_mainstream_content_change][:phone] %></a> at other times</li>
```

Comment:
We want to keep these numbers out of `main`. I suggest dropping the link markup entirely, otherwise you'll need to do something like `<%= foo.sub("0","+44").delete(" ") %>